dev/content-fetcher/squarespace_blog.js

```diff
@@ -0,0 +1,30 @@
+export default class BlogFetcher {
+  constructor(fetch) {
```

Comment:
If you're passing in `fetch` to improve testability, you can also utilise a default parameter `fetch = nodeFetch` so we don't have to pass `node-fetch` in app code, but it can be easily swapped during testing?